recipe/meta.yaml

```diff
@@ -30,6 +30,11 @@ test:
   imports:
     - ParticleSpy
 
+outputs:
```

Comment:
This won't do what you are hoping for.

Instead this should be by placing an empty `__init__.py` file in the Parameters subfolder, as well as appropriate modifications to `MANIFEST.in`, which will ensure this folder gets included in the source tarball and wheel.

Reply:
Thanks for the help. I'll try this instead.

Reply:
Your MANIFEST.in looks actually good, but I think that the Parameters don't get included as you are not including the `ParticleSpy` toplevel folder. 

You can also try something like:

```
recursive-include ParticleSpy/Parameters *
recursive-include ParticleSpy/Data *
```

Reply:
Is this done differently via pip and conda? The directories are included when using pip but not when using conda.

Reply:
There is indeed a difference: with pip you will install the wheel, which has those files included, but conda installs using the source tarball, which doesn't have them.